nilearn/plotting/img_plotting.py

```diff
@@ -738,3 +738,86 @@ def display_factory(display_mode):
                                 **kwargs)
 
     return display
+
+
+def plot_connectome(adjacency_matrix, node_coords,
+                    node_color='auto', node_size=50,
+                    edge_cmap=cm.bwr, edge_threshold=None,
+                    edge_kwargs=None, node_kwargs=None,
+                    output_file=None, display_mode='ortho',
```

Comment:
I would put 'output_file' and 'display_mode', 'title' and 'annotate' before 'edge_kwargs' and 'node_kwargs': I think that these are more important for end-users.